### ISSUE TITLE
to add a multilingual support 

### ISSUE THREAD

**Issue by meghanakn473** [Mohit5Upadhyay/WeatherApplication#10]

**Comment by meghanakn473** [comment]
i want to work on it because i have already worked on it in my project,, Can you please assign this issue to me.

**Comment by Pujitha254** [comment]
Hey Mohith!! just checked your project its pretty good .I like the css part in your project. And we can improve it by adding some features like multilingual options and i wanted to work upon it if it is ok for you.So could you please assign this issue to me.

**Comment by Mohit5Upadhyay** [comment]
Yes , @Pujitha254 & @meghanakn473 you can work on this issue to leverage multilingual support into this project

**Comment by Pujitha254** [comment]
could you please assign it under GSOC extended eddition

**Comment by meghanakn473** [comment]
thank you, can u please assign it under gssoc-ext and hackoctoberfest
